lib/metadata/bucketfile/backend.js

```diff
@@ -17,7 +17,7 @@ import config from '../../Config';
 const errors = arsenal.errors;
 
 const METADATA_PORT = 9990;
-const METADATA_PATH = `${config.filePaths.metadataPath}/`;
+const METADATA_PATH = `${config.filePaths.metadataPath}`;
```

Comment:
Why remove the ```/``` ?
https://github.com/scality/S3/pull/370/files#diff-b0bf037765b0248b6d3dc6e5356cd087R46
```$METADATA_PATH + $ROOT_DB``` = ```$(pwd)/myFavoriteMetadataPathrootDB``` rigth ?